src/bundle/Resources/public/js/scripts/admin.content.edit.js

```diff
@@ -179,6 +182,11 @@
         btn.addEventListener('click', clickHandler, false);
     });
 
-    doc.querySelector('.ibexa-content-edit-content').addEventListener('scroll', fitHeader, false);
+    const contentEditContent = doc.querySelector('.ibexa-content-edit-content');
```

Comment:
isn't there better name? One that avoids word `content` being used in single variable twice?

Reply:
Do you have any propositions? We can also ask @lucasOsti 🙂